hapi/hedera-protobufs/services/token_get_info.proto

```diff
@@ -209,6 +209,31 @@ message TokenInfo {
      * (token definition and individual NFTs).
      */
     Key metadata_key = 28;
+
+    /**
+     * The key which can lock tokens in an account or account partition.
```

Comment:
nitpick:
The summary line should be answering a question, not assigning a purpose.
That is: <dl><dt>What is this?</dt><dd>A thing or subcategory of thing.</dd></dl>
That answer is then followed by any explanatory detail (after a line break).

Here:
```
A function-specific account key.<br/>
This key authorizes transactions to lock tokens in an account or
account partition.
```

Reply:
So to generalize should all fields of type Key follow the same pattern -- all should begin with the same summary line? - in this case - `A function-specific account key.<br/>`

Reply:
All of the function-specific keys (e.g. metadata, kyc, pause, etc...) probably should, yes.
Many summary lines will be similar across messages or fields.  It's the description (after the line break) that gives added detail when the summary is repeated or not sufficiently clear on its own.